src/transformations/tuples.jl

```diff
@@ -13,7 +13,11 @@ geometries wrapping `Tuple` points.
 $APPLY_KEYWORDS
 """
 function tuples(geom, ::Type{T} = Float64; kw...) where T
-    if _is3d(geom)
+    if _ismeasured(geom)
+        return apply(PointTrait(), geom; kw...) do p
+            (T(GI.x(p)), T(GI.y(p)), T(GI.z(p)), T(GI.m(p)))
```

Comment:
It isn't necessary that a geometry has both Z and M.  This would have to be a little more complex (though I feel like we should define some form of framework for this, even if it's just conceptual and in the docs)

Reply:
Hmmmm.... It does say in the [GI docs](https://juliageo.org/GeoInterface.jl/stable/reference/api/#GeoInterface.m-Tuple{Any}) that for a point to be measured, it MUST be a 4-element tuple.

Reply:
That's only for tuple and vector points:

> Length 3 Tuple and Vector points